workflows/VGP-assembly-v2/VGP-Decontamination/VGP-Decontamination-pipeline---Db-not-included-tests.yml

```diff
@@ -0,0 +1,20 @@
+- doc: Test outline for VGP-Decontamination-pipeline---Db-not-included
+  job:
+    Scaffolded assembly (fasta):
+      class: File
+      path: https://www.dropbox.com/s/rl1nkrnicyhgpun/Scaffolded%20assembly%20%28fasta%29.fasta.gz?dl=1
```

Comment:
Wouldn't it be better to put input files on zenodo for example? In case the Dropbox owner clean that file?

Reply:
You're right, I use my dropbox as long as I test the workflows to avoid creating 20 DOI that I'll never use. I'll move them to zenodo

Reply:
Perfect